src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -253,6 +259,70 @@ protected void updateCollectorBatchWithOriginEntries() {
         collectorBatch.setTotalRecords(totalRecords);
         collectorBatch.setTotalAmount(totalDebitAmount);
     }
+    
+  private void consolidateOriginEntries() {

```

Comment:
This new code is a little harder to read when it doesn't use as many spaces. I would suggest modifying this code so that the spacing is in groups of 4 instead of groups of 2, for consistency with the other code in this file.

Reply:
ah, when I was working on checkstyle I was changing these settings, I will set it back and reformat my changes, thanks for noticing